test/fixtures/flow/type-annotations/135/actual.js

```diff
@@ -0,0 +1,3 @@
+type A = {
+	...any,
```

Comment:
Okay was testing on astexplorer, which is a bad idea. Primitives, besides `any`  fail though if I see correctly. https://flowtype.org/try/#0C4TwDgpgBAglC8UDeAoKBIAdNgRgezwBsIBDAOwBo0BfAbiA